tests/integration/test_bundler.py

```diff
@@ -17,7 +17,6 @@
                 packages=({"path": ".", "type": "bundler"},),
```

Comment:
> The `check_vendor_cheksums` parameter has been added to `TestParameters`
> class in commit https://github.com/hermetoproject/hermeto/commit/244114eaa2ebee64575305433166ef333ada9f47.

Unfortunately, the context (aka "why") is completely missing ^here (not your fault) and I'm wondering what's the actual reason it was deemed as a necessary test enhancement. Well, cachito remnant, no context -> let's get rid of it and hopefully some tech debt with it.